infra.bs

```diff
@@ -221,47 +223,51 @@ in parentheses. [[!UNICODE]]
 <div class=example id=example-code-point-notation>
  <p>The <a>code point</a> rendered as 🤔 is represented as U+1F914.
 
- <p>When referring to that <a>code point</a>, we might instead say "U+1F914 THINKING FACE (🤔)",
- instead of just "U+1F914", to provide extra context.
+ <p>When referring to that <a>code point</a>, we might say "U+1F914 (🤔)", to provide extra context.
```

Comment:
I wanted to require either one of them so we keep things brief and consistent. I guess I can be okay with also allowing the very long form though, as long as we don't use it.